packages/docs/components/sidebar/Sidebar.vue

```diff
@@ -1,64 +1,133 @@
 <template>
-  <va-sidebar :minimized="minimized">
-    <template slot="menu">
-      <template v-for="(item, key) in items">
-        <va-sidebar-link-group
-          v-if="item.children"
+  <va-sidebar class="sidebar" color="secondary" v-model="minimized">
+    <va-search />
+    <va-list class="sidebar__links">
+      <va-expand-group v-model="value" multiply>
+        <va-expand
+          v-for="(route, key) in navigationRoutes"
           :key="key"
-          :minimized="minimized"
-          :title="$t(item.displayName)"
-          :children="item.children"
+          class="sidebar__expand"
+          :class="{ 'sidebar__expand--active': value[key] }"
         >
-          <va-sidebar-link
-            v-for="(subMenuItem, index) in item.children"
-            :key="index"
-            :to="subMenuItem.name"
-            :title="$t(subMenuItem.displayName)"
-          />
-        </va-sidebar-link-group>
-        <va-sidebar-link
-          v-else
-          :key="key"
-          :minimized="minimized"
-          :active-by-default="item.name === route.name"
-          :to="item.name"
-        >
-          <span slot="title">{{ $t(item.displayName) }}</span>
-        </va-sidebar-link>
-      </template>
-    </template>
+          <template slot="header">
+            <va-list-item class="sidebar__category">
+              <va-list-item-section class="sidebar__category__section">
+                <va-list-item-label class="sidebar__category__label">
+                  {{ $t(route.displayName) }}
+                </va-list-item-label>
+              </va-list-item-section>
+              <va-list-item-section icon>
+                <va-icon
+                  :name="value[key] ? 'expand_less' : 'expand_more'"
+                  :color="value[key] ? 'primary' : ''"
+                />
+              </va-list-item-section>
+            </va-list-item>
+          </template>
+          <div v-for="(childRoute, index) in route.children" :key="index">
+            <va-list-label
+              v-if="childRoute.category"
+              class="sidebar__subcategory__label"
+              color="gray"
+            >
+              {{ $t(childRoute.category) }}
+            </va-list-label>
+            <va-list-item
+              :to="childRoute.name"
```

Comment:
This doesn't work when you go to "Contribution",  then click on any page on sidebar.